### PR TITLE
change @class in expr for hasclass due to WARNING Error-prone use of …

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
…@class in view

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@hveficent this is failing

File: ao_mrp/views/mrp_production_report.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <template id="report_mrporder"
           inherit_id="mrp.report_mrporder">
-        <xpath expr="//div[@class='row mt32 mb32'][1]" position="inside">
+        <xpath expr="//div[hasclass('row mt32 mb32')][1]" position="inside">
```
Comment (MiquelRForgeFlow) [inline ao_mrp/views/mrp_production_report.xml:6]: -> hasclass('row', 'mt32', 'mb32')

File: ao_purchase/views/report_purchase_order.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
     <template id="report_purchaseorder_document"
           inherit_id="purchase.report_purchaseorder_document">
-        <xpath expr="//div[@class='page']/div[2]/div[1]" position="replace">
+        <xpath expr="//div[hasclass('page')]/div[2]/div[1]" position="replace">
```
Comment (MiquelRForgeFlow) [inline ao_purchase/views/report_purchase_order.xml:6]: remove this as I already did it in https://github.com/Eficent/ao-odoo/pull/71

File: ao_purchase/views/report_purchase_order.xml [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
     <template id="report_purchasequotation_document"
           inherit_id="purchase.report_purchasequotation_document">
-        <xpath expr="//div[@class='page']/div[2]/div[1]" position="replace">
+        <xpath expr="//div[hasclass('page')]/div[2]/div[1]" position="replace">
```
Comment (MiquelRForgeFlow) [inline ao_purchase/views/report_purchase_order.xml:25]: also remove this

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by MiquelRForgeFlow** [comment]
Superseeded by https://github.com/Eficent/ao-odoo/pull/71.

**Pull request closed without merging**